apps/web/src/components/Shared/Attachments.tsx

```diff
@@ -91,7 +91,7 @@ const Attachments: FC<Props> = ({
                 },
                 'relative'
               )}
-              key={url}
+              key={index}
```

Comment:
Do not use index as `key` please change it to url itself, If the url remains the same use `url + index`

Reply:
Thanks for the suggestion, I've updated

Reply:
Thanks for suggestion @harish-sethuraman 🙏🏼 

Do yu have anything else?